index.html

```diff
@@ -9,203 +8,189 @@
+            </div>
+            <div class="mttxt">
+            <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
             </div>
-            <div class="litxt">
+            <button type="button" class="colbtn">Get my CV <img src="./Images/ic_download.png" alt=""></button>
+        </div>
+        <div class="lang_con">
+            <div class="lang">
+                <div class="sectitle">
+                    <h2>Languages</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>JavaScript (ES5/ES6)</li>
                     <li>Ruby</li>
                     <li>HTML</li>
                     <li>CSS</li>
                 </ul>
+                </div>
             </div>
-        </div>
-        <div class="frames">
-            <div class="sectitle">
-                <ul>
-                    <li>
-                        <h2>Frameworks</h2>
-                    </li>
-                </ul>
-            </div>
-            <div class="litxt">
+            <div class="frames">
+                <div class="sectitle">
+                    <h2>Frameworks</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>Bootstrap</li>
                     <li>Ruby on rails</li>
                     <li>RSpec</li>
                     <li>Capibara</li>
                     <li>Selenium</li>
                 </ul>
+                </div>
             </div>
-        </div>
-        <div class="skills">
-            <div class="sectitle">
-                <ul>
-                    <li>
-                        <h2>Skills</h2>
-                    </li>
-                </ul>
-            </div>
-            <div class="litxt">
+            <div class="skills">
+                <div class="sectitle">
+                    <h2>Skills</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>Codekit</li>
                     <li>Codepen</li>
                     <li>GitHub</li>
                     <li>GitLab</li>
                     <li>Terminal</li>
                 </ul>
+                </div>
             </div>
         </div>
     </section>
     <section class="contact">
         <div class="topimg">
         </div>
-        <div class="headline">
-            <h1 class="headtitle" >Get started</h1>
+        <div class="headtitle">
+            <h1 class="mtitle">Get started</h1>
         </div>
-        <div class="maintxt">
+        <div class="mttxt">
             <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
         </div>
         <form action="https://formspree.io/f/mayvvvnd" method="POST">
```

Comment:
- [ ] Kindly make sure your contact section and the footer design stick to the one from the Figma template. Align everything at his expected position by respecting the **spacing**, **width**, **alignment positions**, **fonts**, **backgrounds**, **size**,colors,... Then make sure your design sticks to the Figma template. AS you can see from the images below they look different.
### Figam design
![rf_s4](https://user-images.githubusercontent.com/83042742/171852297-01f5ad87-4cf2-46fe-b70c-4f59044cab1d.png)

### Your design
![rs_S4](https://user-images.githubusercontent.com/83042742/171851865-f52e0db2-b0d4-4f40-86a7-ff2a5701879f.png)